src/Components/Articles.jsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from "react";
+import Article from "./Article";
+
+function Articles() {
+  const [articles, setArticles] = useState([]);
```

Comment:
we need a loading state here to provide visual feedback to the user on slow load